code/datums/materials/basemats.dm

```diff
@@ -33,9 +33,9 @@
 	victim.apply_damage(10, BRUTE, BODY_ZONE_HEAD, wound_bonus = 5, sharpness = TRUE) //cronch
 	return TRUE
 
-/datum/material/glass/on_applied(atom/source, amount, material_flags)
+/datum/material/glass/on_applied_obj(atom/source, amount, material_flags)
 	. = ..()
-	if(isobj(source))
+	if(!isstack(source))
```

Comment:
LOL, I wise man knows when to give up. I respect it.  Although now people will soon post bug reports on why a chair made of glass can break but a glass sheet can't,  probably will be labeled as an oversight.

Reply:
This is a problem I am willing to leave to later me